libsigopt/aux/errors.py

```diff
@@ -0,0 +1,65 @@
+# Copyright © 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache License 2.0
+
+import json
+
+
+class SigoptValidationError(Exception):
```

Comment:
your subclasses should just call `super().__init__(msg)` instead of setting `self.msg` directly

Reply:
A C++ driven obsession with Super() calls being first. Fixed.